ui/store/actions.ts

```diff
@@ -1757,6 +1757,7 @@ export function setSelectedAccount(
       !currentTabIsConnectedToNextAddress;
 
     try {
+      await _setSelectedAddress(address);
```

Comment:
`_setSelectedAddress` function will switch `preferencesController`'s selected address, even though account is not found in `accountsController`, 

it might cause inconsistent address state between `preferencesController` and `accountsController`.

Reply:
To avoid this inconsistency, we will first check if the account is found first, then we will update the controllers.
If account was not found, no controllers will be updated.